packages/xod-cli/package.json

```diff
@@ -3,7 +3,8 @@
   "version": "0.14.0",
   "description": "XOD project: Command Line Interface",
   "scripts": {
-    "build": "babel src/ -d bin/ --source-maps",
+    "copy-workspace": "cpx \"../../workspace/__lib__/**\" \"bin/workspace/__lib__\"",
+    "build": "babel src/ -d bin/ --source-maps && yarn run copy-workspace",
```

Comment:
The common pattern used in other places is having `build:src`, `build:workspace`, and `build` that calls both. Can we use it here as well?

Reply:
Okay, but it's not truth 😬 
`xod-client-electron` contains `build:gui`, `build:app`, `build`, that runs both and runs `copy-workspace` script. This is almost the exact clone (but copies only libraries).
So... rename it in both places into `build:workspace`?
Also, I'm not sure this is a good name, actually we're don't build workspace, we copy it...